integration_tests/e2e/induction/updateInduction.cy.ts

```diff
@@ -61,7 +74,6 @@ context('Update single question in the Induction', () => {
       // Then
       inPrisonWorkPage = Page.verifyOnPage(InPrisonWorkPage)
       inPrisonWorkPage.hasFieldInError('inPrisonWorkOther')
-      // TODO assert the API was not called
```

Comment:
Unfortnuately, this didn't work:

```
cy.wiremockVerify(putRequestedFor(urlEqualTo(`/inductions/${prisonNumber}`)), 0)
```
I can have another look on Monday, but I'm not so bothered about this (we don't do it elsewhere and the main assertion is the actual call to the API above).